src/classes/CRLP_RollupAccount_TEST.cls

```diff
@@ -56,7 +56,7 @@ private class CRLP_RollupAccount_TEST {
     /**
      * @description Create Rollup__mdt records to test rolling up from the Opp and Payment objects to the Account
      */
-    private static void mockRollupCMTValues() {
+    public static void mockRollupCMTValues() {
```

Comment:
I had to change this to public in order to access from the new batch base test class